### PR TITLE
Ft enhancement

### PR DESCRIPTION
**Pull request by hadayda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,22 @@ If you haven't, create a superuser with the [Django createsuperuser command](htt
...
+from drip.signals import post_drip
+
+@receiver(post_drip)
+def handle_drip_sms(sender, **kwargs):
+    drip = kwargs['drip']
+    user = kwargs['user']
+    # Get the message from the drip and the number from the user and handle the SMS message logic.
+```
+- The sms text field supports user object injection, so you can use placeholders for the user data `eg: This sms for {{user.first_name}}` 
+
```
Comment (awartani) [inline README.md:63]: should we add instructions on how to support translation?

File: drip/drips.py [hunk 2]
```diff
@@ -116,7 +135,13 @@ def message(self):
 
             # check if there are html tags in the rendered template
             if len(self.plain) != len(self.body):
-                self._message.attach_alternative(self.body, 'text/html')
+                message_body = self.body.replace(
+                    '<body>',
+                    '<body><span style="display:none;">{}</span>'.format(
+                        self.pre_header
+                    )
+                ) if self.pre_header else self.plain
```
Comment (awartani) [inline drip/drips.py]: this doesn't seem to be right, can you explain what are you trying to accomplish here as `pre_header` should always be in plan text?

File: drip/admin.py [hunk 3]
```diff
@@ -36,6 +39,15 @@ class DripAdmin(admin.ModelAdmin):
     form = DripForm
     users_fields = []
 
+    def get_model_perms(self, request):
+        if not settings.ENABLE_QUERY_SET_RULE_PERMISSION:
+            return super(DripAdmin, self).get_model_perms(request)
+
+        if request.user.groups.filter(name='Drip Admin').exists():
+            return super(DripAdmin, self).get_model_perms(request)
```
Comment (awartani) [inline drip/admin.py:66]: what would happen if it exists?

File: README.md (hunk 1 above)
Comment (hadayda) [reply README.md:63]: Yeah I know I haven't touch the readme yet. I will get back to it

File: drip/drips.py (hunk 2 above)
Comment (hadayda) [reply drip/drips.py]: Yeah you right it doesn't, I was think of doing something then I changed it, but I forgot to remove this.

File: drip/admin.py (hunk 3 above)
Comment (hadayda) [reply drip/admin.py:66]: If the settings is set to enabled, and user doesn't have this Drip Admin group
The Drip will be hidden and at out end we handle the other admin.

**Pull request merged**